### PR TITLE
Blogpost: XMPP Providers Server

### PR DESCRIPTION
**Pull request by Echolon**
CC: @melvo @cal0pteryx

### CODE REVIEW

**Review by melvo — CHANGES_REQUESTED** [review]

File: content/blog/2024-03-28-xmpp-providers-support-chat-server.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+date: 2024-03-28
+---
+
+
+## XMPP Providers Server
+
+We recently started to set up our own XMPP server to provide a [support chat](/blog/2024-03-11-xmpp-providers-chat/).
+Our goal was to automate as much as possible to reduce the maintenance effort to a minimum.
+While doing that, we also thought about how the experience is for XMPP newcomers to set up their own XMPP server.
+There are many [XMPP servers](https://xmpp.org/software/) available, but only few projects focus on quick and simple setups.
```
Comment (melvo) [inline content/blog/2024-03-28-xmpp-providers-support-chat-server.md:12]:
```suggestion

There are many [XMPP servers](https://xmpp.org/software/) available.
But only few projects focus on quick and simple setups.
```

File: content/blog/2024-03-28-xmpp-providers-support-chat-server.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+By using sensible defaults, the project focuses on simplicity rather than on covering all possible use cases.
+With more experience, the setup can be expanded in order to use a more complex configuration.
+
+Read on our [setup of the XMPP Providers Server](https://invent.kde.org/melvo/xmpp-providers-server/-/blob/master/README.md).
+We are happy to hear your feedback!
+
+We encourage you to use XMPP and run your own server instance, maybe even by becoming a public provider.
+But keep in mind that providing a public service comes with great responsibility you should be aware of.
+Best practices can be discussed in many [public chats](https://xmpp.org/community/chat/).
+
```
Comment (melvo) [inline content/blog/2024-03-28-xmpp-providers-support-chat-server.md:30]:
```suggestion
## Help Us

For a good user experience, [apps integrating XMPP Providers](https://providers.xmpp.net/apps/) are as important as the providers themselves.
If you are an XMPP developer, please consider [adding XMPP Provider support](https://invent.kde.org/melvo/xmpp-providers#usage) to your app.
If you are an operator of a public XMPP service, provide the [information we need](https://providers.xmpp.net/faq/#where-do-we-have-the-providers-properties-from) and [add it to our list](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md#providers).

Feel free to [reach out to us](/contact/) if you have any questions!
```

File: content/blog/2024-03-28-xmpp-providers-support-chat-server.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Spread the Word
+
+For a good user experience, [apps integrating XMPP Providers](https://providers.xmpp.net/apps/) are as important as the providers themselves.
+If you are an XMPP developer, please consider [adding XMPP Provider support](https://invent.kde.org/melvo/xmpp-providers#usage) to your app.
+If you are an operator of a public XMPP service, provide the [information we need](https://providers.xmpp.net/faq/#where-do-we-have-the-providers-properties-from) and [add it to our list](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/CONTRIBUTING.md#providers).
```
Comment (melvo) [inline content/blog/2024-03-28-xmpp-providers-support-chat-server.md:35]:
```suggestion
```

**Review by melvo — CHANGES_REQUESTED** [review]

File: content/blog/2024-03-28-xmpp-providers-support-chat-server.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+The XMPP Providers project lives from the community.
```
Comment (melvo) [inline content/blog/2024-03-28-xmpp-providers-support-chat-server.md:37]:
```suggestion
The XMPP Providers project lives from the community.
We are happy to hear your feedback!
```

**Review by Echolon — COMMENTED** [review]
Many thanks!

**Pull request merged**